movement-migration/validation-tool/src/checks/api/matching_feature_flags.rs

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::checks::error::ValidationError;
+use crate::types::api::{MovementAptosRestClient, MovementRestClient};
+use aptos_rest_client::aptos_api_types::{EntryFunctionId, MoveType, ViewRequest};
+use serde_json::json;
+use std::collections::HashSet;
+use std::str::FromStr;
+
+pub struct GlobalFeatureCheck;
+
+impl GlobalFeatureCheck {
```

Comment:
I think you should add some comment to explain what test is done here or a link to the issue that describe the criteria implemented.

Reply:
I think for this migration, we only use latest aptos binary with old db. we don't need to check the feature flag